digdag-docs/src/operators.rst

```diff
@@ -758,7 +758,7 @@ To use Gmail SMTP server, you need to do either of:
       to: [me@example.com]
 
     +step2:
-      _type: mail
+      mail>:
       body: this is email body in string
       subject: workflow started
       to: [me@example.com]
```

Comment:
Should it be `subject`, `body`, or to...
I think current implementation is ok because basically operators take file path at `>: ...`.